### PR TITLE
Many corrections across the site

### PR DESCRIPTION
**Pull request by eatdostacos**
I updated some parts to feel nicer, such as the subheads. Additionally, I found and corrected multiple grammatical inaccuracies over many of the pages.

### CODE REVIEW

File: pages/roadmap.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title : Roadmap
-subhead : The running wishlist and roadmap for the LBRY Foundation🛣️.
+subhead : The wishlist and roadmap for the LBRY Foundation 🛣️
```
Comment (Coolguy3289) [inline pages/roadmap.md:3]: This change should actually be reverted. running wishlist implies that it is constantly being updated, so it is an appropriate term. Not having it there implies a stagnant document unless there is some indication on the page's content that it gets updated

File: pages/tavern.md [hunk 2]
```diff
@@ -7,13 +7,13 @@ layout : default
 permalink : /tavern/
 ---
 
-So you have an idea you want to bring into the LBRY network. You have fleshed out the details and determined its value if it succeeds.
-You know what needs to get done in what order and how you will track completion of your goals... if only you had the resources and help to
-get it rolling!
+So you have an idea that you want to bring into the LBRY network. Have you have fleshed out the details and determined its value if it succeeds?
```
Comment (Coolguy3289) [inline pages/tavern.md:10]: The latter part of this sentence isn't meant to be a question I don't think. These are more persuasion points to show why you would make an initiative.

**Comment by eatdostacos** [comment]
@Coolguy3289 There we go. I changed back the parts you talked about.

**Comment by Coolguy3289** [comment]
@Eniamza This looks good to me. Feel free to merge it if you agree

**Comment by Eniamza** [comment]
Looks good for now

**Pull request merged**